### PR TITLE
CitizenProject & CitizenAction synchronization with API

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: app/config/services/services.xml [hunk 1]
```diff
@@ -171,5 +171,32 @@
 
         <!-- History -->
         <service id="AppBundle\History\EmailSubscriptionHistoryHandler" />
+
+        <!-- Sync with API -->
+
+        <!-- User subscriber -->
+        <service id="AppBundle\Membership\EventListener\UserSubscriber">
+            <argument key="$producer" type="service" id="old_sound_rabbit_mq.sync_api_producer"/>
```
Comment (cacahouwete) [inline app/config/services/services.xml]: It's a question, why not using bind to bind service old_sound_rabbit_mq.sync_api_producer to param name $producer ?

Comment (B-Galati) [reply app/config/services/services.xml]: This is very fragile/error prone for such a service to use bindings. We could quickly make some mistakes without noticing anything.
Without bind It's explicit, easy to read, easy to understand.

Comment (B-Galati) [reply app/config/services/services.xml]: Last thing, we are using `$producer` variable in lots of places but they point to different classes most of the time.

Comment (thomasbisignani) [reply app/config/services/services.xml]: I'm agree with @B-Galati, it's too overkill for this case.

File: features/sync_with_api.feature [hunk 2]
```diff
@@ -1,3 +1,4 @@
+@bonsoir
```
Comment (illusionOfParadise) [inline features/sync_with_api.feature]: Bonsoir? =P

**Pull request merged**